cypress/e2e/tests/pages/users-and-auth/roles.spec.ts

```diff
@@ -131,6 +132,8 @@ describe('Roles Templates', { tags: ['@usersAndAuths', '@adminUser'] }, () => {
       createClusterRole.saveAndWaitForRequests('POST', '/v3/roletemplates').then((res) => {
         const clusterRoleId = res.response?.body.id;
 
+        roleTemplatesToDelete.push(clusterRoleId);
```

Comment:
This was also not a part of the PR but was supporting code that was added by a separate pr in 2.10.